### PR TITLE
Custom date formats

### PR DESCRIPTION
**Pull request by chlirre**
- The DataService's `getData` calls `parse` on a `axisChartMappings` mapping if it exists.
- A custom **date format** can be specified as part of the FsDataProvider configuration. See `dist/app.js`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,9 +18,7 @@
   "author": "Chaaya, Elias & Rashidi, Damoon",
   "license": "MIT",
   "dependencies": {
-    "angular": "^1.5.0",
```
Comment (chaaya) [inline package.json]: Add the removed dependencies!

**Review by chaaya — COMMENTED** [review]
Look at the feedback

**Comment by chlirre** [comment]
@chaaya Deps are back. Will you do the merging?

**Review by chaaya — APPROVED** [review]

**Comment by chaaya** [comment]
great @chlirre! Its merged

**Pull request merged**